DECISION-MAKING.md

```diff
@@ -1,143 +1,154 @@
 `Last update: 2018-07`
 
-# Service owners
+# _Mozilla IAM_ Program Strategy
 
-High level service and product owners for Mozilla IAM.
+* @jeffbryner (IT, InfoSec)
+* @hmitsch (Open Innovation, ParSys)
 
-* Capability owner: @jeffbryner
-  * Product owner: @gdestuynder
+# Mozilla IT Operating Model
+
+* IT Capability Owner: @jeffbryner
+  * IT Product Owner: @gdestuynder
     * Auth0 & Auth0 Lock & Auth0 Dashboard: @jdow @akrug @gdestuynder
+       * <Question: Is `Auth0 Lock` the same as `NLX`>
     * Service Integration (CIS): @akrug
     * LDAP: @jdow
     * DuoSecurity: @gdestuynder
-    * Mozillians.org: @hmitsch
-    * WorkDay: HRIS
-
-## Architects
-
-* @gdestuynder (IAM, Security)
-* @mvk-mozilla (IT)
-
-## Program Managers
-
-* @hmitsch
-
-## User Experience (UX)
-
-* @mbranson
+    
+* IT Enterprise Architecture
+   * @mvk-mozilla
+   
+* Outside-IT Capability Ownership
```

Comment:
Can we call this "Open Innovation" Service Ownership?

Reply:
We can, what should we do with HRIS then? Have an additional item called "People Team" Service Ownership?

Reply:
as per jeff comments below we should probably drop this and drop the IT mention instead